dashcore 0.17.0.0 RC4 - relic fails to build on Fedora 34 (libgcc-11.1.1)

As y'all know, I have been building dash core for years now on Fedora. Fedora 34 was released April 27th.
Building 0.17 fails because, I think anyway, relic is not compatible with gcc-11.1.1? (libgcc-11.1.1-1.fc34.x86_64.rpm)

The error:
```
[ 84%] Building C object _deps/relic-build/src/CMakeFiles/relic_s.dir/md/relic_md_blake2s.c.o
In file included from /builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2s-ref.c:18:
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:101:5: error: size of array element is not a multiple of its alignment
  101 |     blake2s_state S[8][1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:102:5: error: size of array element is not a multiple of its alignment
  102 |     blake2s_state R[1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:109:5: error: size of array element is not a multiple of its alignment
  109 |     blake2b_state S[4][1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:110:5: error: size of array element is not a multiple of its alignment
  110 |     blake2b_state R[1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2s-ref.c: In function 'blake2s':
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2s-ref.c:329:3: error: size of array element is not a multiple of its alignment
  329 |   blake2s_state S[1];
      |   ^~~~~~~~~~~~~
make[3]: *** [_deps/relic-build/src/CMakeFiles/relic_s.dir/build.make:2643: _deps/relic-build/src/CMakeFiles/relic_s.dir/md/blake2s-ref.c.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from /builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/relic_md_blake2s.c:38:
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:101:5: error: size of array element is not a multiple of its alignment
  101 |     blake2s_state S[8][1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:102:5: error: size of array element is not a multiple of its alignment
  102 |     blake2s_state R[1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:109:5: error: size of array element is not a multiple of its alignment
  109 |     blake2b_state S[4][1];
      |     ^~~~~~~~~~~~~
/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/_deps/relic-src/src/md/blake2.h:110:5: error: size of array element is not a multiple of its alignment
  110 |     blake2b_state R[1];
      |     ^~~~~~~~~~~~~
make[3]: Leaving directory '/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build'
make[2]: Leaving directory '/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build'
make[3]: *** [_deps/relic-build/src/CMakeFiles/relic_s.dir/build.make:2656: _deps/relic-build/src/CMakeFiles/relic_s.dir/md/relic_md_blake2s.c.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:237: _deps/relic-build/src/CMakeFiles/relic_s.dir/all] Error 2
make[1]: Leaving directory '/builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build'
make[1]: *** [Makefile:149: all] Error 2
make: *** [funcs.mk:246: /builddir/build/BUILD/dashcore-0.17/dash-0.17.0.0-rc4/depends/work/build/x86_64-redhat-linux-gnu/bls-dash/1.0.1-e24847737f2/build/.stamp_built] Error 2
```

I will continue to experiment, but, that is what the problem looks like to me.

--- a/depends/packages/bls-dash.mk
+++ b/depends/packages/bls-dash.mk
@@ -7,12 +7,12 @@ $(package)_build_subdir=build
 $(package)_sha256_hash=cff7552d8ebd6d29141f1acfa65b7d956c803b575baaaf076d4aea3c32178116
 $(package)_dependencies=gmp cmake
 
-$(package)_relic_version=1885ae3b681c423c72b65ce1fe70910142cf941c
+$(package)_relic_version=3a23142be0a5510a3aa93cd6c76fc59d3fc732a5
 $(package)_relic_download_path=https://github.com/relic-toolkit/relic/archive
 $(package)_relic_download_file=$($(package)_relic_version).tar.gz
 $(package)_relic_file_name=relic-toolkit-$($(package)_relic_download_file)
 $(package)_relic_build_subdir=relic
-$(package)_relic_sha256_hash=951b989b0075212c34f124f508d35391b42f30e14fda9260a68a38c82ebbc032
+$(package)_relic_sha256_hash=ddad83b1406985a1e4703bd03bdbab89453aa700c0c99567cf8de51c205e5dde
 
 $(package)_extra_sources=$($(package)_relic_file_name)
 

